### PR TITLE
GRW-1494 / feat / Add HeroVideoBlock via Storyblok

### PR DESCRIPTION
**Pull request by simonauner**
## Describe your changes

Basically the same PR as https://github.com/HedvigInsurance/racoon/pull/576 but with uploading video assets to Storyblok instead of S3.

image

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1493]: https://hedvig.atlassian.net/browse/GRW-1493?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/store/src/components/HeroVideo/HeroVideo.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+export const HeroVideo = ({
+  sources,
+  poster,
+  height,
+  children,
+  childrenPadding,
+}: HeroVideoProps) => (
+  <HeroVideoWrapper>
+    <StyledVideo playsInline autoPlay muted loop preload="auto" poster={poster} height={height}>
```
Comment (alebedev) [inline apps/store/src/components/HeroVideo/HeroVideo.tsx:56]: Can you add comment explaining why we need muted and playsInline?

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/HeroVideo/HeroVideo.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  }),
+)
+
+const StyledVideo = styled.video(
+  ({ poster, height }: Pick<HeroVideoProps, 'poster' | 'height'>) => ({
+    width: '100%',
+    height: height ?? '80vh',
+    background: `url(${poster}) no-repeat`,
+    backgroundSize: 'cover',
+
```
Comment (guilhermespopolin) [inline apps/store/src/components/HeroVideo/HeroVideo.tsx:43]: nit:
Was the empty line intentional?

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/components/HeroVideo/HeroVideo.tsx (hunk 1 above)
Comment (simonauner) [reply apps/store/src/components/HeroVideo/HeroVideo.tsx:61]: Added 👌

**Pull request merged**